examples/azure/terraform-azure-storage-example/main.tf

```diff
@@ -0,0 +1,56 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A STORAGE ACCOUNT SET
+# This is an example of how to deploy a Storage Account.
+# ---------------------------------------------------------------------------------------------------------------------
+# See test/azure/terraform_azure_storage_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "azurerm" {
+  version = "~> 2.20"
+  features {}
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# PIN TERRAFORM VERSION TO >= 0.12
+# The examples have been upgraded to 0.12 syntax
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  # This module is now only being tested with Terraform 0.13.x. However, to make upgrading easier, we are setting
+  # 0.12.26 as the minimum version, as that version added support for required_providers with source URLs, making it
+  # forwards compatible with 0.13.x code.
+  required_version = ">= 0.12.26"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A RESOURCE GROUP
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_resource_group" "resource_group" {
+  name     = "${var.resource_group_basename}-${var.postfix}"
```

Comment:
set the name to be terratest-rg-${var.postfix} and remove the variable

Reply:
done